todo_app/data/trello_items.py

```diff
@@ -0,0 +1,181 @@
+# Trello card actions
+#
+import requests
+import json
+from todo_app.data.Item_class import Item
+
+_DEFAULT_ITEMS = [
+    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
+    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+]
+
+lists_ids=[]
+items=[]
+
+def get_items(BOARD_ID,APIKey,APIToken):
+    """
+    Fetches all saved items from trello board_id.
+
+    Args:
+        id: The BOARD_ID of the trello board and API_KEY, API_TOKEN.
+
+    Returns:
+        list: The list of cards from a list/board.
+    """
+
+    items=[]
+    query = {
+        'key': APIKey,
+        'token': APIToken,
+        'cards': 'open'
+    }
+
+    url = "https://api.trello.com/1/boards/" + BOARD_ID + "/lists"
+    response = requests.request("GET",url,params=query)
+    jsonResponse = response.json()
+
+    # add try abort response errors i.e. server code 500
+    #
+
+    id_number=1
+    for lists in jsonResponse: 
+        lists_ids.append({'id': lists['id'], 'name': lists['name']})
+        for card in lists['cards']:
+            new_items = Item.from_trello_card(card,lists)
+            newitem = {'id': id_number, 'title': new_items.name, 'status': new_items.status, 'shortLink': new_items.shortLink, 'idList': new_items.idList}
+            items.append(newitem)
+            id_number += 1
```

Comment:
Here you prove that you can use the Item class but then promptly throw away the objects you've just created, so we're back to actually using dictionaries and having that less well defined interface throughout the rest of the program. You're also using custom ids (your id_number) rather than the id provided by Trello which is why I (incorrectly) assumed you could just use the ids directly below - rather than that I'd advise using Trello's ids but displaying things to the user with a more friendly number (see comment on the HTML for how we could do that)

Instead I'd suggest something like:
```suggestion
        for card in lists['cards']:
            new_item = Item.from_trello_card(card,lists)
            items.append(new_item)
```